public/js/myevents.js

```diff
@@ -1,4 +1,581 @@
-require(["jquery", "bootstrap", "auth"], function($) {
-    $("[rel=popover]").popover({container: "body", placement: "left"});
-    $("[title]").not("[rel=popover]").tooltip({container: "body"});
+require([
+   "jquery", "underscore", "backbone", "validate", "client-models", "auth",
+   // plugins
+   "backbone.marionette", "bootstrap", "underscore-template-config"
+], function($, _, Backbone, validate, models, auth) {
+
+$(document).ready(function() { 
+
+	var users = new models.UserList(USER_DATA);
+	var events = new models.EventList(EVENT_DATA);
+
+	var adminsToBeRemoved = [];
+	
+	var modalAdminAdder;
+	var modalAdminRemover;
+	var modalConfirmAdminRemoval;
+
+	var EventRowView = Backbone.Marionette.ItemView.extend({
+	    tagName: 'tr',
+	    template: '#event-row', 
+
+	    events: {
+	        'click .btn-add-admin': 'invokeAddAdminModal',
+	        'click .btn-remove-admin': 'invokeRemoveAdminModal'
+    	},
+
+    	modelEvents: {
+	        'change': 'render'
+	    },	
+
+	    ui: {
+	    	eventAdmins: '.event-admins',
+	    	eventDate: '.event-date',
+	    },
```

Comment:
this removes whitespace in the code: https://github.com/unhangout/unhangout/commit/5de04be686d2ab715f8894cb4f5fb1c87b7f18b7